### ISSUE TITLE
Switching Webpack Config keeps file watches around

### ISSUE THREAD

**Issue by gostay** [mixer/cdk#115]
### Steps to Reproduce

* Open CDK project
* Switch to another webpack config with Run -> Set Webpack Config
* Edit a file referenced by the previous webpack, but not the new one
* Notice the webpack recompile that happens anyway.

### Error

After you switch to a new webpack config, the CDK seems to keep watching files that were referenced by the previous webpack config (in addition to the new ones).

### Editor Details

**Version**: 1.0.0
**Mixer User ID**: 32316796

**Encrypted Editor State**: `{"id":"3ad178dddd4bcf36","password":"R4hIsICmiwje/CgLsErfCw6hZkqPmyAm5aN6JaV0WOc="}`

**Comment by gostay** [comment]
Thanks!  In case it's helpful:  One thing I noticed in my testing was that if I close and restart the CDK, then it only watches the files it needs to for the currently selected webpack config.  It's only when I switch from one to the other that it seems to watch both, which is the weird part.

I should also add that this is not a big deal.  It hardly ever comes up.  And the impact is almost unnoticeable.  It's just something I noticed that I thought you might want to know about.
